config.yaml

```diff
@@ -39,3 +39,12 @@ external_services:
   # Uncomment service_label_name to set up a different label name for grouping all resources of an app version.
   # Default is "version"
   # version_label_name: version
+
```

Comment:
Don't bother updating this file. Its obsolete and out of date.

Reply:
The conversation is resolved but the file is still updated.